### PR TITLE
fix(recurring-job): flooded source label log

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/6257

### CODE REVIEW

File: controller/volume_controller.go [hunk 1]
```diff
@@ -3647,7 +3647,7 @@ func (c *VolumeController) syncPVCRecurringJobLabels(volume *longhorn.Volume) er
 	}
 
 	if !hasSourceLabel {
-		c.logger.Warnf("Ignoring recurring job labels on Volume %v PVC %v due to missing source label", volume.Name, pvc.Name)
+		logrus.Tracef("Ignoring recurring job labels on Volume %v PVC %v due to missing source label", volume.Name, pvc.Name)
```
Comment (innobead) [inline controller/volume_controller.go:3650]: Why not using the existing c.logger?

**Review by innobead — CHANGES_REQUESTED** [review]

File: controller/volume_controller.go (hunk 1 above)
Comment (c3y1huang) [reply controller/volume_controller.go:3650]: [There is no trace in c.logger.](https://github.com/longhorn/longhorn-manager/blob/7a5ca17682b43e6da430d8e459566c713eb4a29f/vendor/github.com/sirupsen/logrus/logrus.go#L139-L177)

Comment (derekbit) [reply controller/volume_controller.go:3650]: Most log messages' levels are corrected. I think we use INFO as default log level. Then, use `c.logger.Debugf` here. WDYT? @c3y1huang @innobead

Comment (innobead) [reply controller/volume_controller.go:3650]: IIRC we still use debug, confirm?

Comment (innobead) [reply controller/volume_controller.go:3650]: @c3y1huang do u know why it doesn't have trace? Duo to a different interface?

Comment (c3y1huang) [reply controller/volume_controller.go:3650]: > @c3y1huang do u know why it doesn't have trace? Duo to a different interface?

Yes, [it got removed due to backward compatibility concerns](https://github.com/sirupsen/logrus/commit/b54cafe5cee93a35bf6b061e261e8265ea0a628c).

Comment (derekbit) [reply controller/volume_controller.go:3650]: > IIRC we still use debug, confirm? @derekbit

How about changing it to INFO from v1.5.1?
I've adjusted most of log messages' levels.

Comment (c3y1huang) [reply controller/volume_controller.go]: Updated.

**Review by derekbit — APPROVED** [review]
LGTM.

BTW, I adjust some logs in https://github.com/longhorn/longhorn-manager/pull/2059.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@Mergifyio backport v1.5.x

**Comment by innobead** [comment]
@c3y1huang need to backport this to 1.4 and 1.3 with a different implementation since their default log level is debug.

**Comment by c3y1huang** [comment]
@Mergifyio backport v1.4.x v1.3.x

**Comment by c3y1huang** [comment]
> @c3y1huang need to backport this to 1.4 and 1.3 with a different implementation since their default log level is debug.

I think we don't need to backport this since the source label is introduced in v1.5.0.

**Comment by innobead** [comment]
@c3y1huang please go ahead to remove backup labels. Thanks.

**Pull request merged**